pages/success.tsx

```diff
@@ -24,8 +25,8 @@ dayjs.extend(toArray);
 dayjs.extend(timezone);
 
 export default function Success(props: inferSSRProps<typeof getServerSideProps>) {
-  const router = useRouter();
-  const { location, name, reschedule } = router.query;
+  const router: NextRouter = useRouter();
+  const { location, name, reschedule }: ParsedUrlQuery = router.query;
```

Comment:
This is inferred so it's obsolete

```suggestion
  const { location, name, reschedule } = router.query;
```